comment/views.py

```
@@ -0,0 +1,199 @@
+        ],
+        responses={200: CommentSerializer(many=True), 404: "Not Found"},
+    )
+    def get(self, request):
+        try:
+            post_id = request.GET.get("post")
+            posts = Post.objects.filter(id=post_id)
+            comments = Comment.objects.filter(post__in=posts)
+            return Response(
+                CommentSerializer(comments, many=True).data, status=status.HTTP_200_OK
+            )
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)